frontend/src/hooks/api/getters/useReleasePlanTemplates/useReleasePlanTemplateInstance.ts

```diff
@@ -0,0 +1,48 @@
+import { useMemo } from 'react';
+import useUiConfig from '../useUiConfig/useUiConfig';
+import { formatApiPath } from 'utils/formatPath';
+import handleErrorResponses from '../httpErrorResponseHandler';
+import { useConditionalSWR } from '../useConditionalSWR/useConditionalSWR';
+import { useUiFlag } from 'hooks/useUiFlag';
+import type { IReleasePlanTemplateInstance } from 'interfaces/releasePlans';
+
+const path = (templateId: string) =>
+    `api/admin/release-plan-templates/${templateId}`;
+
+const DEFAULT_DATA: IReleasePlanTemplateInstance = {
+    id: '',
+    name: '',
+    description: '',
+    milestones: [],
+    createdAt: '',
+    createdByUserId: 0,
+};
+
+export const useReleasePlanTemplateInstance = (templateId: string) => {
```

Comment:
I assume this hook is needed to grab detailed data about the release plan template? Since we don't return all data in the GET all endpoint?

If so maybe this is more consistent with what we usually do?
```suggestion
export const useReleasePlanTemplate = (templateId: string) => {
```

Reply:
Yeah fair point. The issue is we have a useReleasePlanTemplates (plural). I can maybe rename that in a different PR